packages/clerk-js/src/ui/elements/Drawer.tsx

```diff
@@ -416,6 +420,9 @@ interface ConfirmationProps {
   onOpenChange: (open: boolean) => void;
   children: React.ReactNode;
   actionsSlot: React.ReactNode;
+  /**
+   * @see https://floating-ui.com/docs/userole
+   */
```

Comment:
🙃 Intentional?
```suggestion
```

Reply:
It is yeah, I can add in a separate commit if you'd like. But just thought i'd add a link to the docs similar to what we're doing at the root component.

Reply:
That's fine, just wanted to make sure.